language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ClientResourceAccessActionNodeContext.java

```diff
@@ -106,8 +106,13 @@ public List<LSCompletionItem> getCompletions(BallerinaCompletionContext context,
                 List<LSCompletionItem> items = this.getCompletionItemList(exprEntries, context);
                 completionItems.addAll(items);
             } else {
-                completionItems.addAll(this.actionKWCompletions(context));
-                completionItems.addAll(this.expressionCompletions(context));
+                List<Node> arguments = new ArrayList<>();
+                node.arguments().ifPresent(argList -> 
+                        arguments.addAll(argList.arguments().stream().collect(Collectors.toList())));
+                if (!isInNamedArgOnlyContext(context, arguments)) {
```

Comment:
Since we're using the negation of isInNamedArgOnlyContext most of the time, shall we invert it?